### PR TITLE
Add inverse scaling

### PR DESCRIPTION
**Pull request by Sjoerd1993**
Add inverse scaling to the amount of options. Needed some tweaks to get this to work correctly with panning/zooming, `select_data`, as well as with axis-dependent operations like `shift_vertically` and `optimize_limits`.

Finally, this also adds an `optimize_limits` call to scaling changes, as the scaling is often not good at all directly after changing scale, so running `optimize_limits` is often the first thing you want to do anyway.

### CODE REVIEW

**Comment by Sjoerd1993** [comment]
> Finally, this also adds an optimize_limits call to scaling changes, as the scaling is often not good at all directly after changing scale, so running optimize_limits is often the first thing you want to do anyway.

Given PR #440, I may want to remove that change anyway.

File: data/ui/preferences.blp
Comment (cmkohnen) [inline data/ui/preferences.blp:1]: I think your're somehow using an old version of this file here

Comment (Sjoerd1993) [reply data/ui/preferences.blp:1]: Thanks for noticing, this is correct. I know that I started this as a local branch (before the scaling got removed from preferences), somehow this got out of sync.

**Comment by cmkohnen** [comment]
Correct me if I'm wrong, but now the new scale isn't represented in preferences anymore

**Comment by Sjoerd1993** [comment]
> Correct me if I'm wrong, but now the new scale isn't represented in preferences anymore

Yes, but all scales have been moved away from preferences since PR #435, so that should be the case.

**Comment by cmkohnen** [comment]
Oh sorry, my bad

**Review by cmkohnen — APPROVED** [review]

**Pull request merged**